LocalSettings.php

```diff
@@ -3956,6 +3956,22 @@
 			'css' => [],
 		],
 	],
+	
+	// TextExtracts
+	'wgExtractsRemoveClasses' => [
+		'default' => false,
+		'gratispaideiawiki' => [
+			'.metadata', // This is a good thing to remove in any case
+			'span.coordinates',
+			'span.geo-multi-punct',
+			'span.geo-nondefault',
+			'#coordinates',
+		],
+	],
+	'wgExtractsExtendOpenSearchXml' => [
+		'default' => null,
```

Comment:
This isn't the extension default: https://github.com/wikimedia/mediawiki-extensions-TextExtracts/blob/master/extension.json#L61

Reply:
Done

Reply:
Please see my comment on the other PR. The default needs to match the one in extension.json AND you need to add a different value for your wiki if you need one.